internal/npm_install/package.json

```diff
@@ -2,6 +2,8 @@
     "description": "runtime dependencies for npm_install",
     "devDependencies": {},
     "dependencies": {
-        "removeNPMAbsolutePaths": "1.0.4"
+        "removeNPMAbsolutePaths": "1.0.4",
+        "source-map": "0.5.7",
```

Comment:
why are these added? it feels like a workaround for some issue - I don't see why npm_install cares about source maps. Can we change the place where this is require'd to make it non-fatal instead?

Reply:
Removed these in favor of using @build_bazel_rules_nodejs_node_loader_deps//:node_modules for the source-map-support require in node_loader.js